launcher/ui/dialogs/ModUpdateDialog.cpp

```diff
@@ -158,8 +158,8 @@ void ModUpdateDialog::checkCandidates()
             if (!reason.isEmpty())
                 text += tr("Reason: %1").arg(reason) + "<br>";
             if (!recover_url.isEmpty())
-                text += tr("Possible solution: ") + tr("Getting the latest version manually:") + "<br>" +
-                        QString("<a href='%1'>").arg(recover_url.toString()) + recover_url.toString() + "</a><br>";
+                text += tr("Possible solution: Getting the latest version manually:<br>"
+                           "<a href='%1'>%1</a><br>").arg(recover_url.toString());
```

Comment:
ehm i don't really like putting the HTML tags inside the translation string, they are never translated so it's just more manual work on the translator's part :/

Reply:
I think we do that in most of the codebase, so it is probably fine?
I personally don't care much about this when translating to German

Reply:
I think it's slightly annoying :tm:
no need to change it if you don't want to